.ci_support/win_64_.yaml

```diff
@@ -1,7 +1,3 @@
-c_stdlib:
-- vs
```

Comment:
Weird. Re-rendering shouldn't have removed this

Note that this is what leads to the CI error for Windows:

```
conda_libmamba_solver.conda_build_exceptions.ExplainedDependencyNeedsBuildingError: Unsatisfiable dependencies for platform win-64: {MatchSpec("c_win-64")}
Encountered problems while solving:
  - nothing provides requested c_win-64
```

Will try re-rendering again

Reply:
On closer inspection, it looks like re-rendering with conda-smithy is doing the right thing

The `cuda-nvtx` package has a `skip` for `win-64`. As a result is ignored when generating the variants. Hence `c_stdlib` is dropped

However for some reason it is still being considered by `conda-build` when building `cuda-nvtx-dev`